api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirTaskServiceImpl.java

```diff
@@ -131,4 +135,22 @@ public Task updateTask(String uuid, Task task) {
 		return dao.searchForTasks(basedOnReference, ownerReference, status, sort).stream().map(translator::toFhirResource)
 		        .collect(Collectors.toList());
 	}
+
+	@Override
+	public Task updateTask(String uuid, Task task) {
+		// TODO Auto-generated method stub
+		return null;
+	}
```

Comment:
This *should* rely on the `BaseFhirService`'s `update()` method, with the tests changed to call that instead.

Reply:
Seems am breaking same class

Reply:
Yeah... so this will also need to change in the interface and the place where the interface is used.